### PR TITLE
Mm/easier setup

### PR DESCRIPTION
**Pull request by MarcMcIntosh**
Adds `lerna bootstrap` to the top level post install script, this makes it easier to install everything in one command.
Also removed the call to `lerna bootstrap` from the ci as it's no longer needed

## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] All new and existing tests are passing.

### CODE REVIEW

**Review by Miniplop — APPROVED** [review]
+1 Don't forget to mention the new command behavior to the other dev :)

File: packages/next-slicezone/package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "build": "rollup -c",
     "prepublishOnly": "yarn build && yarn test",
     "lint": "eslint --ext .js,.jsx .",
-    "security-audit:prod": "yarn audit --production --only=prod --audit-level=low",
+    "security-audit:prod": "yarn audit --groups dependencies --audit-level=high",
```
Comment (Miniplop) [inline packages/next-slicezone/package.json:13]: I don't think this is the way to solve this problem I think. This line mean that we will ignore moderate security issues.

**Review by Miniplop — CHANGES_REQUESTED** [review]
I don't think hiding the problem is a good solution. Which moderate issues are hard to fix or introduce regressions ? What are the options for these libs ?

**Comment by MarcMcIntosh** [comment]
closed in favour of https://github.com/prismicio/slice-machine/pull/185

**Pull request closed without merging**